roles/kubernetes-apps/network_plugin/multus/tasks/main.yml

```diff
@@ -9,9 +9,12 @@
     state: "latest"
   delegate_to: "{{ groups['kube_control_plane'][0] }}"
   run_once: true
-  with_items: "{{ multus_manifest_1.results }} + {{ multus_nodes_list|map('extract', hostvars, 'multus_manifest_2')|list|json_query('[].results') }}"
+  with_items: "{{ multus_manifest_1.results + multus_nodes_list|map('extract', hostvars, 'multus_manifest_2.results')|list }}"
```

Comment:
sorry that didnt work.
```
 "msg": "Failed to template loop_control.label: 'None' has no attribute 'item'. 'None' has no attribute 'item'"
```

I could not get it to work otherwise, so I added a task after this one to fail if this task resulted with item.name 'None'

Reply:
i even tried setting
```
default([{'item': {'name': 'None', 'type': 'None', 'file': 'None'}}]
```

Reply:
Hmmm actually I am thinking that the extract might simply not works... Does this works better?
```suggestion
  with_items: "{{ multus_manifest_1.results + multus_nodes_list|map('extract', hostvars, 'multus_manifest_2')|list|json_query('[].results') }}"
```

Reply:
sorry i cant test locally in the next day or two.... CI is failing too.. will push again later to retry

Reply:
No worries, thanks for all your OSS works :heart: